src/omni_epd/displays/inky_display.py

```diff
@@ -48,111 +52,71 @@ def __init__(self, deviceName, config):
         elif(dType == 'what'):
             deviceObj = self.load_display_driver(self.pkg_name, 'what')
             self._device = deviceObj.InkyWHAT(dColor)
+        elif(dType == 'impression'):
+            deviceObj = self.load_display_driver(self.pkg_name, 'inky_uc8159')
+            self._device = deviceObj.Inky(dColor)
+            self.mode = dColor = 'color' # default to color for impression
         elif(dType == 'auto'):
             deviceObj = self.load_display_driver(self.pkg_name, 'auto')
             self._device = deviceObj.auto()
             dColor = 'color' if self._device.colour == 'multi' else self._device.colour
 
+        self.clear_color = deviceObj.WHITE
+
         # set mode to black + any other color supported
         if(self.mode != "black"):
             self.modes_available = ('black', dColor)
 
-        # phat and what devices expect colors in the order white, black, other
+        # phat and what devices expect colors in the order white, black, other, 
         if(self.mode == "red" and dColor == "red"):
             self.palette_filter.append([255, 0, 0])
             self.max_colors = 3
         elif(self.mode == "yellow" and dColor == "yellow"):
             self.palette_filter.append([255, 255, 0])
             self.max_colors = 3
+        elif(self.mode == "color" and dColor == "color"):
+            self.palette_filter = deviceObj.DESATURATED_PALETTE
+            self.max_colors = 8
+            self.clear_color = deviceObj.CLEAN

```

Comment:
```suggestion
            from inky.inky_uc8159 import DESATURATED_PALETTE
            self.palette_filter = DESATURATED_PALETTE
            self.max_colors = 8
```
`DESATURETED_PALETTE` doesn't exist in `auto.py`, so we have to import it from `inky_uc8159.py` (or include it ourselves) for this to work with `inky.auto`